openwisp_utils/admin_theme/templates/admin/openwisp_info.html

```diff
@@ -15,4 +15,29 @@ <h2>{% trans "OS Information" %}</h2>
 <p><strong>{% trans "OS version" %}:</strong> {{ system_info.os_version }}</p>
 <p><strong>{% trans "Kernel version" %}:</strong> {{ system_info.kernel_version }}</p>
 <p><strong>{% trans "Hardware platform" %}:</strong> {{ system_info.hardware_platform }}</p>
+
+{% block metric_collect_consent %}
+    {% if metric_collection_installed %}
+    <h2>Metric collection</h2>
+        <form method="POST" id="id_metric_collection_consent_form">
+            {% csrf_token %}
+            {{ metric_consent_form }}
```

Comment:
Please rewrite this so that we can have the input first and each line wrapped on a `<p>` element, the current output is not readable.

You probably need to add:

```
    margin-right: 8px;
    vertical-align: top;
```
To the checkbox input to make it look decent:

![Screenshot from 2024-04-16 11-52-42](https://github.com/openwisp/openwisp-utils/assets/841044/f796d015-d9c5-4991-96b4-f7e39cd2d48e)

Reply:
I just used some CSS to achieve the same result. Re-writing this would require copying some code from Django which we will need to maintain over time.